### DISCUSSION TITLE
Simple custom script never ends

### DISCUSSION THREAD

**Discussion by SThomas986** [netbox-community/netbox#15928] [Help Wanted!]
I've started trying to learn to write some custom scripts for our Netbox instance, no matter what I include in the script it just sits and says "Results pending..." forever and ever and ever. The script never exits. I've created a bare bones script that does absolutely nothing but return a string, and same results.

Here's the code as it stands today (many things tried, but this is it as is stands):

```
from extras.scripts import *

name = 'Hello World Module'

class Hello_World(Script):

    class Meta:
        name = 'Hello World Script'
        description = 'Hello World'
        scheduling_enabled = False
        commit_default = False

    def run(self, data, commit):
        try: 
            self.log_success('Hello Success')
            return '\n'.join('Hello World')
        except:
            self.log_failure('hello error')
            return '\n'.join('error')
```

**Comment by candlerb** [comment]
Is the netbox-rq service running?

**Comment by hackcl** [comment]
If you go back to the Custom Scripts Overview Page, does it show a last run? Experienced similar issue but it showed the run finished on the main page...

**Comment by oneCrazyAdmin** [comment]
I have a similar issue. I am able to run the script using the CLI, but not the web interface.  The CLI runs as the admin user (local) but the web interface runs as my LDAP user.

When I run "journalctl -u netbox-rq -n 100", I end up with a message
         AttributeError: Can't get attribute 'NBLDAPBackend' on <module 'netbox.authentication' from '/opt/netbox/netbox/netbox/authentication/__init__.py'>

Not sure if you are using LDAP, but this was working before moving to version 4.0.6 from 3.x

Scripts do run correctly in the web interface with a user that is not logging in via LDAP
